### PR TITLE
FAT-6788 Added test cases for EDGPATRON-114

### PR DESCRIPTION
**Pull request by pavankumar181**
## Purpose
The Purpose is this PR is to cover the implementation of EDGPATRON-114 with karate test cases.

## Approach
Added test case scenarios which are required for EDGPATRON-114

### TODOS and Open Questions
- [ ] Use GitHub checklists. When solved, check the box and explain the answer.

### Learning
_Describe the research stage. Add links to blog posts, patterns, libraries or addons used to solve this problem._

### CODE REVIEW

**Review by nirmalr-shah — APPROVED** [review]

File: edge-patron/src/test/java/org/folio/EdgePatronTests.java [hunk 1]
```diff
@@ -8,7 +8,6 @@
 import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
-@Disabled
```
Comment (gurleenkaurbp) [inline edge-patron/src/test/java/org/folio/EdgePatronTests.java:11]: revert this

File: edge-patron/src/main/resources/prokopovych/edge-patron/features/patrons.feature [hunk 2]
```diff
@@ -44,12 +44,12 @@ Feature: patron tests
     * call read('classpath:prokopovych/edge-patron/features/util/initData.feature@PostCheckOut')
 
     Given url edgeUrl
-    And path 'patron/account/' + extSystemId+ '?includeLoans=true'
+    And path 'patron/account/' + extSystemId
     And param apikey = apikey
+    And param query = 'includeLoans=true'
```
Comment (gurleenkaurbp) [inline edge-patron/src/main/resources/prokopovych/edge-patron/features/patrons.feature:49]: Please revert the code from the scenarios where your change is not there.

File: edge-patron/src/main/resources/prokopovych/edge-patron/features/patrons.feature [hunk 3]
```diff
@@ -61,12 +61,12 @@ Feature: patron tests
...
 
     Given url edgeUrl
-    And path 'patron/account/' + extSystemId + '?includeCharges=true'
+    And path 'patron/account/' + extSystemId
     And param apikey = apikey
+    And param query = 'includeLoans=true'
     When method GET
     Then status 200
     And match response.totalChargesCount == 1
-    And match response.charges[0].item.itemId == itemId
```
Comment (gurleenkaurbp) [inline edge-patron/src/main/resources/prokopovych/edge-patron/features/patrons.feature:69]: Please revert the code from the scenarios where your change is not there.

Comment (pavankumar181) [reply edge-patron/src/main/resources/prokopovych/edge-patron/features/patrons.feature]: reverted

**Review by gurleenkaurbp — APPROVED** [review]

**Review by Vignesh-kalyanasundaram — APPROVED** [review]

**Pull request merged**